0.5.1b firmware update

MrPrattASH opened the pull request:
2 major changes to this repo since the latest 0.5.1b huskylens firmware

1. Several methods no longer return "Knocked Received" upon successful completion. These return values have been updated to return blank. There is presently no logic present to confirm that the changes have been made as the husky lens returns a blank byte array. Rest assured, the methods still do indeed change algorithms, display text, etc, they simply have no return values.
2. I've updated the read.me to reflect these changes in return values.

MrPrattASH committed:
0.5.1b firmware update

2 major changes to this repo since the latest 0.5.1b huskylens firmware

1. Several methods no longer return "Knocked Received" upon successful completion. These return values have been updated to return blank. There is presently no logic present to confirm that the changes have been made as the husky lens returns a blank byte array. Rest assured, the methods still do indeed change algorithms, display text, etc, they simply have no return values.
2. I've updated the read.me to reflect these changes in return values.

hktan125 commented:
@MrPrattASH Thanks for contribution.
I havent maintained this repository for the last few years, and is not actively working on it anymore, and I will not be reviewing or actively updating it.
However, I appreciate your PR. Feel free to fork it and continue working on it if you're interested.

MrPrattASH committed:
Sequential Read Request

Add small delay to setting methods, to prevent skipped returns on multiple setting commands sequentially in quick succession.
